ui/src/app/images/components/UbuntuImageSelect/ArchSelect/ArchSelect.tsx

```diff
@@ -4,14 +4,15 @@ import { useSelector } from "react-redux";
 
 import type { ImageValue } from "app/images/types";
 import type {
+  BaseImageFields,
   BootResourceUbuntuArch,
   BootResourceUbuntuRelease,
 } from "app/store/bootresource/types";
 import configSelectors from "app/store/config/selectors";
 
 type Props = {
   arches: BootResourceUbuntuArch[];
-  release: BootResourceUbuntuRelease | null;
+  release: BootResourceUbuntuRelease | BaseImageFields | null;
```

Comment:
I've had to update a lot of the `UbuntuImageSelect` props to work with data coming from both `ubuntu.releases` and `fetchedImages.releases`. I think this is a bug in the backend as they should be the same, i.e both have the key `unsupported_arches`

Reply:
Might be worth leaving a comment about when you get the different types.

Reply:
Yeah good call